package.json

```diff
@@ -2,8 +2,7 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "dev-prod": "env-cmd -f .env.production next dev",
-    "dev": "env-cmd -f .env.development next dev",
+    "dev": "next dev --port 5000",
```

Comment:
Next.js does not read `PORT` environment variable from `.env` out of box. The argument will be soon removed anyone, once the API endpoint is switched to a different port (e.g. 4000...4999)